src/components/TextArea/stories.scss

```diff
@@ -0,0 +1,7 @@
+.showcaseTextArea {
```

Comment:
I think it's better to put such styles inside `storybook` directory, since sources of component directory are bundled, and it might be that this file will be added to build dir as well

Reply:
Hmm, its not referenced in the index file - its only imported through the *.stories.tsx file. I don't think it will be added to the bundle, since the *.stories.tsx is not

Reply:
Yes, you are right, I just checked and it's not bundled, true